spring-cloud-gcp-docs/src/main/asciidoc/logging.adoc

```diff
@@ -23,16 +23,15 @@ https://cloud.google.com/logging/[Stackdriver Logging] is the managed logging se
 Platform.
 
 This module provides support for associating a web request trace ID with the corresponding log entries.
-This allows grouping of log messages by request.
+It does so by retrieving the `X-B3-TraceId` value from the https://logback.qos.ch/manual/mdc.html[Mapped Diagnostic Context (MDC)], which is set by Spring Cloud Sleuth.
+If Spring Cloud Sleuth isn't used, the configured `TraceIdExtractor` extracts the desired header value and sets it as the log entry's trace ID.
+This allows grouping of log messages by request, for example, in the https://console.cloud.google.com/logs/viewer[Google Cloud Console Logs viewer].
 
-NOTE: Due to the way logging is set up, the GCP project ID and credentials defined in
-`application.properties` are ignored.
-Instead, you should set the `GOOGLE_CLOUD_PROJECT` and `GOOGLE_APPLICATION_CREDENTIALS` environment
-variables to the project ID and credentials private key location, respectively.
+NOTE: Due to the way logging is set up, the GCP project ID and credentials defined in `application.properties` are ignored.
+Instead, you should set the `GOOGLE_CLOUD_PROJECT` and `GOOGLE_APPLICATION_CREDENTIALS` environment variables to the project ID and credentials private key location, respectively.
+You can do this easily if you're using the http://cloud.google.com/sdk[Google Cloud SDK], using the `gcloud config set project [YOUR_PROJECT_ID]` and `gcloud auth application-default login` commands, respectively.
 
-`TraceIdLoggingWebMvcInterceptor` extracts the request trace ID from an HTTP request using a
-`TraceIdExtractor` and stores it in a thread-local of the https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/google-cloud-logging/src/main/java/com/google/cloud/logging/TraceLoggingEnhancer.java[`TraceLoggingEnhancer`]
-which can then be used in a logging appender to add the trace ID metadata to log messages.
+`TraceIdLoggingWebMvcInterceptor` extracts the request trace ID from an HTTP request using a `TraceIdExtractor` and stores it in a thread-local, which can then be used in a logging appender to add the trace ID metadata to log messages.
```

Comment:
If you make it conditional on Sleuth missing, then you should add that note here too.